keps/sig-network/5495-deprecate-ipvs-mode-in-kube-proxy/kep.yaml

```diff
@@ -0,0 +1,36 @@
+title: Deprecate ipvs mode in kube-proxy
+kep-number: 5495
+authors:
+  - "@adrianmoisey"
+owning-sig: sig-network
+participating-sigs: []
+status: provisional
+creation-date: 2025-08-25
+reviewers:
+  - thockin
+  - danwinship
+  - aojea
+approvers:
+  - thockin
+  - danwinship
+  - aojea
+
+see-also:
+  - "/keps/sig-network/265-ipvs-based-load-balancing/kep.yaml"
```

Comment:
also KEP-3986 (nftables), which talks about deprecating IPVS

oh, and people usually link to the README.md (or the directory) not the kep.yaml, I think